### ISSUE TITLE
Download pretrained model

### ISSUE THREAD

**Issue by ma-siddiqui** [SpaceNetChallenge/RoadDetector#5]
How to download s3://spacenet-dataset/SpaceNet_Roads_Competition/Pretrained_Models/01-Albu/weights/

It required an account with credit card details. So I need to pay to download this? Please clarify and let me know how can I get it.

Thanks,
Muhammad Ajmal Siddiqui

**Comment by josemarcosrf** [comment]
@ma-siddiqui  - I believe was moved to: s3://spacenet-dataset/spacenet-model-weights/spacenet-3/01-Albu/weights and seems to be free to donwload

**Comment by ignaciolasa** [comment]
Yes it is free! Did you achieve to load the downloaded models? Because I have this issue:

"ModulError: pytorch_zoo.unet not found"
